libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ProbabilityDistributionBoxChart.tsx

```diff
@@ -46,7 +46,7 @@ export class ProbabilityDistributionBoxChart extends React.Component<IProbabilit
             cohort,
             index,
             this.props.probabilityOption?.key || "",
-            this.props.probabilityOption?.id,
+            this.props.probabilityOption?.text,
```

Comment:
We should try avoid using text as input paramter, since it is user facing values, subject to change at anytime.
Is it possible to use some sort of map to get string values based on the key?
If key is not used anywhere, we can also change the key to be string.

Reply:
Agreed. See my comment as well.

Reply:
The SDK side has little idea as to what id value is. The text value is a column value that SDK backend for large data recognizes. There are a lot of UI jargons that we translate for SDK like feature names for cohorts etc. This is yet another example.

I agree that we should have clearer contracts with SDK backend.